network/transport/tcp_socket.c

```diff
@@ -0,0 +1,158 @@
+/***************************************************************************//**
```

Comment:
I think we should almost never have a commit body empty. Just a short meta description of the file like what it does and why it needs to do it.

Reply:
done